hooks/hooks.py

```diff
@@ -1,18 +1,46 @@
 #!/usr/bin/python
+
+import setup
+setup.pre_install()
+import consul
 import json
 import os
 import subprocess
 import sys
 
 from charmhelpers.core import hookenv, host
+from charmhelpers import fetch
+from path import path
 
 
 hooks = hookenv.Hooks()
 
-CONF_PATH = "/etc/consul.json"
-BIN_PATH = "/usr/local/bin/consul"
+CONF_PATH = '/etc/consul.json'
```

Comment:
Good catch Chuck!  I only changed the quotes to be consistent on that one, tried **not** to inject path.py on things I was not actively changing.  I can do that on the next iteration.